fem/intrules.hpp

```diff
@@ -203,13 +203,7 @@ class IntegrationRule : public Array<IntegrationPoint>
    IntegrationRule() : Array<IntegrationPoint>() { }
 
    /// Construct an integration rule with given number of points
-   explicit IntegrationRule(int NP) : Array<IntegrationPoint>(NP)
-   {
-      for (int i = 0; i < this->Size(); i++)
-      {
-         (*this)[i].Init();
-      }
-   }
+   explicit IntegrationRule(int NP) : Array<IntegrationPoint>(NP) { }
```

Comment:
The `Array` class constructor used here does not perform any initialization, that is why there was explicit initialization here. What was your reasoning behind the removal of the initialization?